src/utilities/ToastQueueManager.ts

```diff
@@ -21,20 +21,24 @@ class ToastQueueManager {
         this.isToastVisible = false;
     }
 
-    show(error: AxiosError<ServerErrorResponse>, options?:ToastOptions, message?:string): any {
+    show(error?: AxiosError<ServerErrorResponse>, options?:ToastOptions, message?:string): any {
         const ToastConfig = {...DefaultToastConfig, options}
```

Comment:
If I understand this correctly, in order to display a custom message one must `show(undefined, undefined, 'My Message')`


**Can we either:**
1. Separate method called `showMessage(message, options?)`
    1. Maybe rename to `showServerError(...)`
    
2. Alternatively, use de-structured named parameters which are then all optional.  
```
show({ error, options, message }: { error?: AxiosError<ServerErrorResponse>; options?: ToastOptions; message?: string }):void { ... }
```